core/tests/performance_framework/perf_services.py

```diff
@@ -148,11 +203,14 @@ def get_page_timings_from_cached_session(self, page_url):
         """
```

Comment:
I meant, would combining tests speed things up? For example, instead of four tests per page, you could have two tests per page, for instance:

    (a) get uncached and cached timings (b) get uncached and cached size

or you could also split it up as:

    (a) get uncached timings and size  (b) get cached timings and size

or you could combine them all together, reducing four tests to one:

    get uncached metrics (timings + size), then reload again to get cached metrics

Is there a reason for not doing this combination?